features/build/build_controller.rb

```diff
@@ -27,7 +27,7 @@ class BuildController < AuthenticatedControllerBase
         project: project,
         build: build,
         title: "Project #{project.project_name}, Build #{build.number}",
-        existing_rows: current_build_runner.all_build_output_log_lines.collect { |a| a[:html] }
+        existing_rows: current_build_runner.all_build_output_log_rows.collect(&:html)
```

Comment:
I kinda think we should standardize on `map` instead of `collect`. `map` is more widely known.

Reply:
`collect` is English though

Reply:
It is english language, but in my experience, because `map` is more commonly found in *programming languages* vs `collect`, it might be better to go with since it seems more universal.